detectors/node/opentelemetry-resource-detector-container/test/ContainerDetector.test.ts

```diff
@@ -146,5 +146,75 @@ describe('ContainerDetector', () => {
       sinon.assert.calledOnce(readStub);
       assertEmptyResource(resource);
     });
+
+    describe(' extractContainerId from line tests', () => {
+      const containerDetector = new ContainerDetector();
+      it('should extract container ID from crio-prefixed line', () => {
+        const line =
+          '11:devices:/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pod5c5979ec_6b2b_11e9_a923_42010a800002.slice/crio-1234567890abcdef.scope';
+        const expected = '1234567890abcdef';
+        assert.strictEqual(
+          containerDetector['extractContainerIdFromLine'](line),
```

Comment:
Accessing the private / protected function this way feels wrong. 
WDYT about extracting the function to `utils.ts` instead of it being class member function. then we can test it directly in an idiomatic way.

Reply:
it does feel wrong. I shall move the helper methods as per suggestion.